src/test/java/seedu/address/logic/commands/ViewDetailsCommandTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.item.NameIsExactlyPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code ViewDetailsCommand}.
+ */
+public class ViewDetailsCommandTest {
+
```

Comment:
is it possible to test results of command? eg. if no such item exists, or check that the item being shown is indeed the right item